state/database.go

```
@@ -94,6 +96,7 @@ func NewDatabase(stateDB, codeDB dbm.DB) (*Database, error) {
 	db.ethTrieDB = ethtrie.NewDatabase(&core.EthereumDB{CodeDB: codeDB})
 
 	db.codeSizeCache, _ = lru.New(codeSizeCacheSize)
```

Can we not ignore the errors here of either of these line and just return the error?